netket/operator/_discrete_operator_jax.py

```diff
@@ -181,12 +181,15 @@ def n_conn(self, x, out=None) -> np.ndarray:
 
         Returns:
             array: The number of connected states x' for each x[i].
-
         """
+        _, mels = self.get_conn_padded(x)
```

Comment:
in the hope of getting jax to do some extra CSE, this function should be jitted.

Reply:
mmm like a lambda function with jit decorator?

Reply:
yeah jut a jax.jit. Don't worry, I pushed it myself.